### PR TITLE
Updating Readmes I forgot to update.

### PR DESCRIPTION
**Pull request by vputys**

### CODE REVIEW

File: board_specific_readmes/rpi4b.md
Comment (pywtk) [inline board_specific_readmes/rpi4b.md:1]: The instructions aren't as copy-pastable as they could be. With some refinements someone should be able to replicate the build quicker.

File: board_specific_readmes/rpi4b.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Raspberry Pi 4 Model B 
+
+***Note: `kirkstone` yocto branch***
+
+- Clone these layers to your work directory:
+``` 
+    git clone git://git.yoctoproject.org/poky.git -b kirkstone
```
Comment (pywtk) [inline board_specific_readmes/rpi4b.md]: add " && \" to the end of all but the last of these lines so should an error occur the later lines do not execute.

File: board_specific_readmes/rpi4b.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    git clone git://git.openembedded.org/meta-openembedded  -b kirkstone
+    git clone git://git.yoctoproject.org/meta-raspberrypi.git -b kirkstone
+```
+- Source work environment - `source <your_path>/poky/oe-init-build-env`
```
Comment (pywtk) [inline board_specific_readmes/rpi4b.md]: without being in a '```' comment block, this instruction doesn't benefit from the copy icon in the top right hand corner.

Comment (pywtk) [inline board_specific_readmes/rpi4b.md]: by putting in the "<your-path>" users are going to have to edit every line they copy.

A different approach would be to suggest they make a working directory first & then assume every instruction executes from there.

File: board_specific_readmes/rpi4b.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+- Add following layers to your `bblayers.conf` file usually found in `<your_path>/build/conf/`:
+```
+    <your_path>/meta-openembedded/meta-oe \
```
Comment (pywtk) [inline board_specific_readmes/rpi4b.md]: replace these with a single bitbake-layer call i.e. 

`bitbake-layers add-layer \
../meta-raspberrypi \
../meta-iotconnect \
../meta-myExampleIotconnectLayer`

File: board_specific_readmes/rpi4b.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    <your_path>/meta-openembedded/meta-oe \ 
+    <your_path>/meta-openembedded/meta-python \ 
+    <your_path>/meta-openembedded/meta-networking \ 
+    <your_path>/meta-openembedded/meta-multimedia \
+    <your_path>/meta-raspberrypi \
+    <your_path>/iotc-yocto-c-sdk/meta-iotconnect \
+    <your_path>/iotc-yocto-c-sdk/meta-myExampleIotconnectLayer 
+```
+
+- Change `MACHINE` variable in `local.conf` file to `MACHINE = "raspberrypi4"`
```
Comment (pywtk) [inline board_specific_readmes/rpi4b.md]: change these to `echo -n "stuff" >> conf/local.conf`

File: board_specific_readmes/rpi4b.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+- Add these lines to the same `local.conf` file:
+```
```
Comment (pywtk) [inline board_specific_readmes/rpi4b.md]: very interested to learn why these are required?

File: board_specific_readmes/rpi4b.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    DISTRO_FEATURES:append = " systemd"
+    VIRTUAL-RUNTIME_init_manager = "systemd"
+
+    LICENSE_FLAGS_ACCEPTED = " synaptics-killswitch"
+```
+- Optionally add `ENABLE_UART = "1"`
+
+- Add IoTC C SDK to your image (in your build image .bb file or in `local.conf`) - `IMAGE_INSTALL += " iotc-c-sdk"`
```
Comment (pywtk) [inline board_specific_readmes/rpi4b.md]: change to `echo`

File: board_specific_readmes/rpi4b.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+- Build your image - `bitbake <your_image>`
```
Comment (pywtk) [inline board_specific_readmes/rpi4b.md]: change to `core-image-minimal` or similar

File: board_specific_readmes/rpi4b.md
Comment (pywtk) [reply board_specific_readmes/rpi4b.md:1]: perhaps even putting this at the end as a "copy & paste & it should just work"


```
git clone git://git.yoctoproject.org/poky.git -b kirkstone && \
git clone git://git.openembedded.org/meta-openembedded  -b kirkstone && \
git clone git://git.yoctoproject.org/meta-raspberrypi.git -b kirkstone && \
wget https://github.com/vputys/iotc-yocto-c-sdk/archive/refs/heads/kirkstone.zip -O/tmp/tmp.zip && \
unzip /tmp/tmp.zip -d /tmp/ && \
mv /tmp/iotc-yocto-c-sdk-kirkstone/meta-* ../ && \
rm -r /tmp/tmp.zip /tmp/iotc-yocto-c-sdk-kirkstone && \
source poky/oe-init-build-env && \
bitbake-layers add-layer \
../meta-raspberrypi \
../meta-iotconnect \
../meta-myExampleIotconnectLayer && \
echo -n 'MACHINE="raspberrypi3"' >> conf/local.conf && \
echo -n 'INHERIT += "extrausers"
EXTRA_USERS_PARAMS = " useradd -p '\$5\$qjMzQ4sWI8S\$AJ/zzrYE2PJoyM9e6QLOV.L/xyDn0Lmk2E/aH4wd7o.' iot; \
                       usermod  -a -G sudo iot;"
IMAGE_INSTALL += " openssh"
IMAGE_INSTALL:append = " iotc-c-sdk"' >> conf/local.conf && \
bitbake core-image-minimal
```

**Review by pywtk — CHANGES_REQUESTED** [review]
great stuff! some changes requested & one query

File: board_specific_readmes/rpi4b.md (hunk 5 above)
Comment (vputys) [reply board_specific_readmes/rpi4b.md]: me and Alen had problems with build/run without them

**Review by pywtk — APPROVED** [review]

**Pull request merged**